packages/datetime2/src/index.ts

```diff
@@ -48,4 +52,8 @@ export {
     type TimezoneSelectProps,
     /** @deprecated import from `@blueprintjs/datetime` instead */
     TimezoneDisplayFormat,
+    /** @deprecated import from `@blueprintjs/datetime` instead */
+    TimePicker,
+    /** @deprecated import from `@blueprintjs/datetime` instead */
+    type TimePickerProps,
 } from "@blueprintjs/datetime";
```

Comment:
Changed: https://github.com/palantir/blueprint/pull/6868/commits/3669f3e6519cb2184a7867435b72bcf7e2ec2b07

Reply:
I think the deprecation warnings in datetime to use the v3 components are because the old components themselves are deprecated. The deprecation warnings here are to push users to import from `datetime` when available, because (I imagine) that is ultimately the package that the v3 components will be promoted to.

Ultimately it doesn't matter too much what we do here since this is just a deprecation warning, but I think leaving the deprecation warning for all of these re-exports is the right thing to do since if followed it will lead to less churn when ultimately updating to the next major blueprint version.

Reply:
I see, didn't realize that these components specific to `datetime2` would be brought into `datetime`

Reply:
That's my understanding, though I am new to working on Blueprint and we may end up doing something else. Thank you for updating to keep the same pattern for now!